packages/react-router-dom/rollup.config.js

```diff
@@ -26,13 +26,7 @@ module.exports = function rollup() {
         sourcemap: !PRETTY,
         banner: createBanner("React Router DOM", version),
       },
-      external: [
-        "history",
-        "react",
-        "react-dom",
-        "react-router",
-        "@remix-run/router",
-      ],
+      external: ["react", "react-dom", "react-router", "@remix-run/router"],
```

Comment:
`react-router-dom` no longer depends on `history` so it can be removed from `external` and `globals`